plugins/workspace-backpack/src/backpack.js

```diff
@@ -578,7 +578,8 @@ export class Backpack extends Blockly.DragTarget {
    * @protected
    */
   isOpenable_() {
-    return !this.isOpen();
+    return !this.isOpen() &&
+    this.options_.contextMenu.allowEmptyBackpackOpen? true: this.getCount() > 0;
```

Comment:
Surprised the linter didn't complain about this, but could you add a space before the `?` and `:` please: https://google.github.io/styleguide/jsguide.html#:~:text=On%20both%20sides%20of%20any%20binary%20or%20ternary%20operator.

and then this line will probably need to be wrapped to stay under 80 chars